pkg/controller/controller_instance.go

```diff
@@ -1426,6 +1426,9 @@ func (c *controller) pollServiceInstance(instance *v1beta1.ServiceInstance) erro
 		if deleting {
 			toUpdate.Status.DeprovisionStatus = v1beta1.ServiceInstanceDeprovisionStatusSucceeded
 		}
+		if !deleting {
```

Comment:
s/if !deleting/else/

Reply:
The reason that I didn't use else here is that the two branches are not related. It is just coincidental that the conditions are opposites.